ocaml/stdlib/camlinternalOO.ml

```diff
@@ -16,12 +16,22 @@
 
 open! Stdlib
 
-[@@@ocaml.flambda_o3]
-
 [@@@ocaml.inline 0]
 [@@@ocaml.afl_inst_ratio 0]
 
-open Obj
+let magic x = Sys.opaque_identity (Obj.magic x)
+let obj x = Sys.opaque_identity (Obj.obj x)
```

Comment:
Maybe call this of_repr or unrepr or something? `obj` is an extremely common variable name in the code below and it's confusing.

Reply:
Fixed